yt/visualization/line_plot.py

```diff
@@ -222,7 +222,7 @@ def from_lines(
         figure_size : int or two-element iterable of ints
             Size in inches of the image.
             Default: 5 (5x5)
-        fontsize : int
+        font_size : int
```

Comment:
I agree with the annotated type, but the default value's type should also reflect it, can you fix it too ?

Reply:
Should be good now!